actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "WebSocket error occurred: #{message}"
```

Comment:
In my case it is something I was looking for and wasn't there (an error regarding WebSockets). It's hard to say regarding other cases.